src/VisibleComponent/VisibleComponent.example.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: basic.html
+title: VisibleComponent HOC example
+description: This example shows the usage of the VisibleComponent HOC (High Order Component).
+collection: Examples
+---
+
+This example shows the usage of the VisibleComponent HOC (High Order Component) to
+determine the visibility of a component based on a `activeModules` property. Typically
+this property is managed globally by `react-redux` (or similiar).
```

Comment:
A little bit more prose would be nice. E.g. we define three compononents, but because of the HOC we only see 2… OK?

Reply:
Check.